ui/component/chat/chatComment/view.jsx

```diff
@@ -0,0 +1,289 @@
+// @flow
+import 'scss/component/_livestream-comment.scss';
+
+import { getStickerUrl } from 'util/comments';
+import { Menu, MenuButton } from '@reach/menu-button';
+import { parseURI } from 'util/lbryURI';
+import * as ICONS from 'constants/icons';
+import ChannelThumbnail from 'component/channelThumbnail';
+import classnames from 'classnames';
+import CommentBadge from 'component/common/comment-badge';
+import CommentMenuList from 'component/commentMenuList';
+import CreditAmount from 'component/common/credit-amount';
+import DateTime from 'component/dateTime';
+import Empty from 'component/common/empty';
+import Icon from 'component/common/icon';
+import MarkdownPreview from 'component/common/markdown-preview';
+import OptimizedImage from 'component/optimizedImage';
+import React from 'react';
+import PremiumBadge from 'component/premiumBadge';
+import { Lbryio } from 'lbryinc';
+
+type Props = {
+  comment: Comment,
+  forceUpdate?: any,
+  uri: string,
+  isMobile?: boolean,
+  chatMode?: string,
+  restoreScrollPos?: () => void,
+  handleCommentClick: (any) => void,
+  handleDismissPin?: () => void,
+  // --- redux:
+  claim: StreamClaim,
+  stakedLevel: number,
+  claimsByUri: { [string]: any },
+  odyseeMembership: string,
+  activeChannelClaim?: any,
+  authorTitle: string,
+  channelAge?: any,
+};
+
+export const ChatCommentContext = React.createContext<any>();
```

Comment:
_I think you already mentioned something about provider, but logging it so don't miss._

Still need to wrap with the provider, otherwise it breaks embed in livestreams.
http://localhost:9090/@dev-test:8/ls-1:e (embeds should bring you to the page, now play in a weird floating way).